plugins/feeds/public/obtemoslab_urls.py

```diff
@@ -0,0 +1,37 @@
+import requests
+import logging
+from datetime import datetime, timedelta
+from core.observables import Url
+from core.feed import Feed
+from core.errors import ObservableValidationError
+from core.config.config import yeti_config
+
+class Obtemoslab(Feed):
+
+    default_values = {
+        "frequency": timedelta(hours=24),
+        "name": "Obtemoslab",
+        "source" : "http://tracker.0btemoslab.com/tracker/Malware.txt",
+        "description": "List of payload locations",
+    }
+
+    def update(self):
+        resp = requests.get(self.source, proxies=yeti_config.proxy)
+        if resp.ok and resp.content:
+            lines = resp.content.split("\r\n")[4:-1]
+            for url in lines:
+                self.analyze(url)
+
+    def analyze(self, url):
+
+        context = {}
+        context['date_added'] = datetime.now()
+        context['source'] = self.name
+
+        try:
+            url_data = Url.get_or_create(value=url)
+            url_data.normalize()
+            url_data.add_context(context)
```

Comment:
Adding a context with `datetime.now()` will create a new `context` entry every single time that feed runs. You may want to either remove the `date_added` from there (and maybe even the context altogether?)

Also, I notice you never tag observables in your feed - IMO this could be tagged with `payload_delivery`